### PR TITLE
Merging Code Reviews intro 

### PR DESCRIPTION
**Pull request by xtreger**
Can you review the intro for the task and let me know if I can go ahead and merge it to the task branch.

### CODE REVIEW

**Review by oisinallen99 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
-Software Quality Assurance Assignment
+## Code Reviews
+
+
+Code reviews are a very important part of the software development process which involves analysing
+the source code to identify bugs. A code review process is typically conducted 
+before merging with the codebase.
+
+Code reviews improve code quality making your codebase more stable. They prevent bugs and errors
+from getting into your project at an early stage of the software development process.
```
Comment (diyarenny) [inline README.md:9]: Only one suggestion maybe introduce the points that will be covered in the code review section of the handbook.

**Review by JouharaAlwardi — APPROVED** [review]
I think the introduction for code reviews looks good.

File: README.md (hunk 1 above)
Comment (xtreger) [reply README.md:9]: Yeah, I'll do that now thanks!

**Review by diyarenny — APPROVED** [review]
I'm happy with the introduction, approving PR.

**Pull request merged**